container/Dockerfile

```diff
@@ -1,15 +1,30 @@
 # BUILDER
 FROM quay.io/centos/centos:stream9 AS builder
 
+# Add ARGs to enable or disable the Demo config
+# Set this argument to any value to enable the Demo config
+# Demo mode is enabled by appending --build_arg _EN_DEMO=true to the build command
+# Demo mode is disabled by ommiting the --build_arg option
+ARG _EN_DEMO
+RUN if [[ -z "$_EN_DEMO" ]] ; \
+	then \
+		echo "Demo is disabled" ; \
+	else \
+		echo "Demo is enabled" ;\
+	fi
+
 # build ARGs
 ARG _RADARE2_GIT_REPO="https://github.com/radareorg/radare2.git"
 ENV RADARE2_GIT_REPO=$_RADARE2_GIT_REPO
 # Radare2 constitutes a noteworthy dependency. 
 # However, it's a software that undergoes consistent changes across commits. 
 # In my experience, even within the same minor version, significant alterations can be observed. 
 # Despite ks-nav not utilizing any uncommon features of Radare2, I prefer to adhere to the most thoroughly tested commit.
-ARG _RADARE2_GIT_COMMIT=4a07098809c71c21515d0f862ac4ca044ae95e95
-ENV RADARE2_GIT_COMMIT=$_RADARE2_GIT_COMMIT
+# ARG _RADARE2_GIT_COMMIT=4a07098809c71c21515d0f862ac4ca044ae95e95
+# ENV RADARE2_GIT_COMMIT=$_RADARE2_GIT_COMMIT
+# The following version has also been tested
+ARG _RADARE2_GIT_REPO_TAG="5.9.0"
+ENV RADARE2_GIT_REPO_BRANCH=$_RADARE2_GIT_REPO_TAG
```

Comment:
Need to remove commented out lines and break this tag into radare version bump ticket.